### ISSUE TITLE
Replace all symlinks by copies

### ISSUE THREAD

**Issue by konstantinfickel** [fsmatheaugsburg-team/markdown-static-site-generator#10]
I'd like to open the discussion about replacing all symlinks by copies of the files.

When our webspace was moved by the Rechenzentrum, the symlinks bekame invalid, but where not replaced by the system because they were already there. Additionally, the sacrifice of small webspace would be worth it relying on less os features and making it easier to move our installation around.

**Comment by AntonLydike** [comment]
I don't think this justifies hard copies... I think links benefit the simplicity and intuitiveness. You can remove files from the public folders and see the update immediately without rebuilding.... But I am open for a discussion.
